docker-compose.yml

```diff
@@ -67,7 +67,7 @@ services:
       Sitecore_Sitecore__IdentityServer__CertificateRawData: ${SITECORE_ID_CERTIFICATE}
       Sitecore_Sitecore__IdentityServer__PublicOrigin: https://${ID_HOST}
       Sitecore_Sitecore__IdentityServer__CertificateRawDataPassword: ${SITECORE_ID_CERTIFICATE_PASSWORD}
-      Sitecore_License: ${SITECORE_LICENSE}
+      # Sitecore_License: ${SITECORE_LICENSE}
```

Comment:
Could you please add a `# DEMO TEAM CUSTOMIZATION - Configure for a mounted license file instead of using SITECORE_LICENSE` comment line above each of the `SITECORE_LICENSE` commented lines?

This helps us when comparing with newer Sitecore versions and merging files during upgrades.